backend/stakepoold/signal.go

```diff
@@ -19,13 +19,13 @@ var shutdownSignaled = make(chan struct{})
 // withShutdownCancel creates a copy of a context that is cancelled whenever
 // shutdown is invoked through an interrupt signal or from an JSON-RPC stop
 // request.
-func withShutdownCancel(ctx context.Context) context.Context {
-	ctx, cancel := context.WithCancel(ctx)
+func withShutdownCancel(spd context.Context) context.Context {
```

Comment:
I think this should have remained `ctx context.Context`?

Reply:
Good catch.